SwrveSDK/src/main/java/com/swrve/sdk/ISwrveBase.java

```diff
@@ -40,6 +40,8 @@
 
     void userUpdate(Map<String, String> attributes);
 
+    void userUpdateDate(String name, Date date);
```

Comment:
Now that the other api is gone, it would be nicer if it was just userUpdate(String name, Date date) ?
(ie change name from userUpdateDate to userUpdate)